db/migrate/20170303023308_create_exams.rb

```diff
@@ -0,0 +1,13 @@
+class CreateExams < ActiveRecord::Migration[5.0]
+  def change
+    create_table :exams do |t|
+      t.string :score
+      t.datetime :exam_date
```

Comment:
It's probably cleaner to change this line than to add a new migration below since this isn't merged yet, but it's not a huge deal. Just less confusing in the future.